Softeq.XToolkit.WhiteLabel/Bootstrapper/Containers/DryIocContainerBuilder.cs

```diff
@@ -69,39 +74,40 @@ public void RegisterBuildCallback(Action<IContainer> action)
 
         public IContainer Build()
         {
-            Singleton<DryIocContainer, IContainer>();
+            var container = new DryIocContainerAdapter();
 
-            var container = _container.Resolve<IContainer>();
-            _container = _container.WithNoMoreRegistrationAllowed();
+            _dryContainer.RegisterInstance<IContainer>(container, IfAlreadyRegistered.Keep);
```

Comment:
I wander if something like this could be possible:

in constructor of DryIocContainerAdapter
```
public DryIocContainerAdapter(DryIoc.IContainer container)
{
    container.RegisterInstance<IContainer>(this, IfAlreadyRegistered.Keep);
    _container = container.WithNoMoreRegistrationAllowed();
}
```

so Initialize() method would become redundant and here it would be just:
`var container = new DryIocContainerAdapter(_dryContainer);`

Reply:
Thanks for the detailed review!
Looks good, but in this case, we can't afford that, because `DryIocContainerBuilder` needs the latest (locked) instance of` DryIoc.IContainer`, and I would like to keep the adapter as simple as possible.